docs/Using Randomness/chainlink-vrf.md

```diff
@@ -4,30 +4,108 @@ section: ethereum
+Use Chainlink VRF to build reliable smart contracts for any applications that rely on unpredictable outcomes:
+
+- Building blockchain games and NFTs.
+- Random assignment of duties and resources. For example, randomly assigning judges to cases.
+- Choosing a representative sample for consensus mechanisms.
+
+For help with your specific use case, [contact us]() to connect with one of our Solutions Architects. You can also ask questions about Chainlink VRF on [Stack Overflow](https://stackoverflow.com/questions/ask?tags=chainlink). <!-- TODO: Add contact URLs -->
+
+## Supported networks
+
+Chainlink VRF v2 is currently available on the following networks:
+
+- Ethereum:
+  - [Mainnet](/docs/vrf-contracts/#ethereum-mainnet)
+  - [Rinkeby testnet](/docs/vrf-contracts/#rinkeby-testnet)
+- Binance Smart Chain
+  - [Testnet](/docs/vrf-contracts/#binance-smart-chain-testnet)
+- Polygon (Matic):
+  - [Mumbai testnet](/docs/vrf-contracts/#polygon-matic-mumbai-testnet)
+
+See the [Contract Addresses](/docs/vrf-contracts) page for a complete list of coordinator addresses and gas price limits.
+
+To learn when VRF v2 becomes available on more networks, follow us on [Twitter](https://twitter.com/chainlink) or sign up for our [mailing list](/docs/developer-communications/).
+
+## Subscriptions
+
+VRF v2 requests receive funding from subscription accounts. The [Subscription Manager](https://vrf.chain.link) lets you create an account and pre-pay for VRF v2 so you don't provide funding each time your application requests randomness. This reduces the total gas cost to use VRF v2. It also provides a simple way to fund your use of Chainlink products from a single location so you don't have to manage multiple wallets across several different systems and applications.
+
+Subscriptions have the following core concepts:
+
+- **Subscription accounts:** An account that holds LINK tokens and makes them available to fund requests to Chainlink VRF v2 coordinators.
+- **Subscription owner:** The wallet address that creates and manages a subscription account. Any account can add LINK to the subscription balance, but only the owner can add approved consumers or withdraw funds.
+- **Consumers:** Contracts that are approved to use funding from your subscription account.
+- **Subscription balance:** The amount of LINK maintained on your subscription account. Requests from consumer contracts will continue to be funded until the balance runs out, so be sure to maintain sufficient funds in your subscription balance to pay for the requests and keep your applications running.
+
+The Subscription Manager system has the following limitations:
+
+- Each subscription supports up to 100 consumer contracts. If you need more than 100 consumers, create multiple subscriptions.
+- Each subscription must maintain a minimum balance in order to fund requests from consumer contracts. If your balance drops below this threshold
```

Comment:
@dwightjl this is an incomplete sentence.

Reply:
Good catch. I moved the "request will go through automatically as long as the request was made in the last 24 hours" sentence up from the Subscription billing section to resolve this.